sw/device/lib/dif/dif_base.c

```diff
@@ -0,0 +1,22 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_base.h"
+
+#include <stdbool.h>
+
+bool toggle_to_bool(dif_toggle_t val) {
```

Comment:
Can you give these a `dif` prefix? Also worth moving them into the header and marking them `inline` (and leaving extern decls in the .c file for linking purposes).

Reply:
Done.